invokeai/frontend/web/src/features/system/components/AboutModal/AboutModal.tsx

```diff
@@ -0,0 +1,140 @@
+        <InvModalContent maxH="80vh" h="80vh">
+          <InvModalHeader>{t('accessibility.about')}</InvModalHeader>
+          <InvModalCloseButton />
+          <InvModalBody display="flex" flexDir="column" gap={4}>
+            <ScrollableContent>
+              <Grid templateColumns="repeat(2, 1fr)">
+                <GridItem backgroundColor="base.750" borderRadius="base">
+                  <TableContainer>
+                    <Table variant="unstyled" w="50%">
+                      <Thead>
+                        <Tr>
+                          <Th fontSize="medium" color="white">
+                            {t('common.localSystem')}
+                          </Th>
+                        </Tr>
+                      </Thead>
+                      {appDeps_arr.map((deps, index) => (
+                        <Tbody key={index}>
+                          {Object.entries(deps).map(([key, value]) => (
+                            <Tr key={key} fontSize="sm" color="white">
+                              <Td py={2}>{key}</Td>
+                              <Td py={2}>{value ? value : 'Not Installed'}</Td>
+                            </Tr>
+                          ))}
+                        </Tbody>
+                      ))}
+                    </Table>
+                  </TableContainer>
+                </GridItem>
+                <GridItem>
+                  <Flex flexDir="column" gap={3} mt="5rem" alignItems="center">
+                    <Image
+                      ref={ref}
+                      src={InvokeLogoYellow}
+                      alt="invoke-logo"
+                      w="120px"
+                      h="80px"
+                      minW="24px"
+                      minH="24px"
+                      userSelect="none"
+                    />
+                    <InvText>
+                      {t('common.v')}
+                      {appVersion?.version}
+                    </InvText>
+                    <Grid templateColumns="repeat(2, 1fr)" gap="3">
+                      <GridItem>
+                        <Link fontSize="sm" href={githubLink} isExternal>
+                          {t('common.githubLabel')}
+                          <ExternalLinkIcon mx="2px" />
+                        </Link>
+                      </GridItem>
+                      <GridItem>
+                        <Link fontSize="sm" href={discordLink} isExternal>
+                          {t('common.discordLabel')}
+                          <ExternalLinkIcon mx="2px" />
+                        </Link>
+                      </GridItem>
+                    </Grid>
+                    <InvHeading fontSize="large">
+                      {t('common.aboutHeading')}
+                    </InvHeading>
+                    <InvText fontSize="sm">{t('common.aboutDesc')}</InvText>
+                    <Link href={websiteLink} fontSize="sm">
+                      {t('common.websiteLink')}
+                    </Link>
```

Comment:
Use `websiteLink` as the text. There's no need to use translation here.

Also, this is an external link.